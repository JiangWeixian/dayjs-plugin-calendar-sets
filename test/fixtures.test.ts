import dayjs from 'dayjs'
import CalendarSets from '../src/index'

dayjs.extend(CalendarSets)

it('fix #2, wrong date calendar sets', () => {
  const sets = dayjs.calendarSets({ year: 2023 }).month({ month: 0 })
  expect(sets).toMatchSnapshot()
})
