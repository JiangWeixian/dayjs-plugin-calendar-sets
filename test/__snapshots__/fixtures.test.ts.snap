// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fix #2, wrong date calendar sets 1`] = `
Array [
  Array [
    "",
    "",
    "",
    "",
    "",
    "",
    "2023-01-01",
  ],
  Array [
    "2023-01-02",
    "2023-01-03",
    "2023-01-04",
    "2023-01-05",
    "2023-01-06",
    "2023-01-07",
    "2023-01-08",
  ],
  Array [
    "2023-01-09",
    "2023-01-10",
    "2023-01-11",
    "2023-01-12",
    "2023-01-13",
    "2023-01-14",
    "2023-01-15",
  ],
  Array [
    "2023-01-16",
    "2023-01-17",
    "2023-01-18",
    "2023-01-19",
    "2023-01-20",
    "2023-01-21",
    "2023-01-22",
  ],
  Array [
    "2023-01-23",
    "2023-01-24",
    "2023-01-25",
    "2023-01-26",
    "2023-01-27",
    "2023-01-28",
    "2023-01-29",
  ],
  Array [
    "2023-01-30",
    "2023-01-31",
    "",
    "",
    "",
    "",
    "",
  ],
]
`;
